src/me/EtienneDx/RealEstate/Transactions/ClaimAuction.java

```diff
@@ -110,10 +112,16 @@ else if(!Utils.makePayment(owner, null, price, false, false))
                     if(getHolder().getState() instanceof Sign)
                     {
                         Sign sign = (Sign) getHolder().getState();
-                        sign.setLine(0, "");
-                        sign.setLine(1, Messages.getMessage(RealEstate.instance.messages.msgSignAuctionWon, false));
-                        sign.setLine(2, buyerPlayer.getName());
-                        sign.setLine(3, "");
+                        SignSide front = sign.getSide(Side.FRONT);
+                        SignSide back = sign.getSide(Side.BACK);
+                        front.setLine(0, "");
+                        front.setLine(1, Messages.getMessage(RealEstate.instance.messages.msgSignAuctionWon, false));
+                        front.setLine(2, buyerPlayer.getName());
+                        front.setLine(3, "");
+                        back.setLine(0, "");
+                        back.setLine(1, Messages.getMessage(RealEstate.instance.messages.msgSignAuctionWon, false));
+                        back.setLine(2, buyerPlayer.getName());
+                        back.setLine(3, "");
```

Comment:
if you have some time, would be cool to have something like setLine(front, back, "stuff"). Otherwise that works (there's already a bazillion things to refactor haha ^^')

Reply:
Done.  I created a RealEstateSign class that encapsulates this and "isRealEstateSign" functionality.